oak/server/rust/oak_runtime/src/node/logger.rs

```diff
@@ -58,13 +54,15 @@ impl super::Node for LogNode {
         let runtime = self.runtime.clone();
         let node_id = self.node_id;
         let reader = self.reader;
-        // TODO(#770): Use `std::thread::Builder` and give a name to this thread.
-        let thread_handle = spawn(move || {
-            let pretty_name = format!("{}-{:?}:", config_name, thread::current());
-            let result = logger(&pretty_name, runtime.clone(), node_id, reader);
-            info!("{} LOG: exiting log thread {:?}", pretty_name, result);
-            runtime.remove_node_id(node_id);
-        });
+        let thread_builder = thread::Builder::new().name(format!("node_logger={}", config_name));
+        let thread_handle = thread_builder
+            .spawn(move || {
+                let pretty_name = format!("{}-{:?}:", config_name, thread::current());
```

Comment:
The `Debug` implementation of `Thread` is not great, how about making it a bit better by introducing a separate function that can make it prettier? It may show the thread id and optional thread name in a more compact name than the current one. Feel free to create a separate issue and we can follow up there perhaps.

Reply:
Done.

Also used in the runtime, see the example in the commit message.